www/js/services.js

```diff
@@ -15,15 +15,35 @@ angular.module('emission.services', ['emission.plugin.logger',
             window.cordova.plugins.BEMServerComm.getUserPersonalData("/profile/create", successCallback, errorCallback);
     };
 
+    var processErrorMessages = function(errorMsg) {
+      var titleMsg = "Error"
+      if (errorMsg.includes("403")) {
+        titleMsg = "Error: OPcode not found";
```

Comment:
why do you need the titleMsg? Doesn't seem to be used anywhere.

Reply:
Removed in https://github.com/e-mission/e-mission-phone/pull/914/commits/a590bf6409a61d4f0a3cb999079cce9302c4a99b